test/pipelinerun_test.go

```diff
@@ -172,6 +174,14 @@ func TestPipelineRun(t *testing.T) {
 			if len(events) != td.expectedNumberOfEvents {
 				t.Fatalf("Expected %d number of successful events from pipelinerun and taskrun but got %d; list of receieved events : %#v", td.expectedNumberOfEvents, len(events), events)
 			}
+			pvc, err := c.KubeClient.Kube.CoreV1().PersistentVolumeClaims(namespace).Get(artifacts.GetPVCName(pipelineRun), metav1.GetOptions{})
```

Comment:
maybe include a comment here explaining what this block of code is testing?

Reply:
Done.